bindings/per-isolate-data.hh

```diff
@@ -8,6 +8,7 @@ namespace dd {
 
 class PerIsolateData {
  private:
+  v8::Isolate* isolate_;
```

Comment:
The cleanup hook knows what isolate it is being called on ? Should we store this ?

Reply:
The callback doesn't receive it, and it's non-capturing.